src/main/kotlin/uk/gov/justice/digital/hmpps/pollpush/services/health/QueueHealth.kt

```diff
@@ -50,10 +50,7 @@ class QueueHealth(@Autowired @Qualifier("awsSqsClient") private val awsSqsClient
         MESSAGES_IN_FLIGHT.healthName to queueAttributes.attributes[MESSAGES_IN_FLIGHT.awsName]?.toInt()
     )
 
-    val health = Health.Builder().up().withDetails(details).addDlqHealth(queueAttributes).build()
-
-    log.info("Found health details for queue '{}': {}", queueName, health)
```

Comment:
this information is already available and graphed, no need to be output here for each health check